### PR TITLE
fix: Theming Support #45

### LINKED ISSUE #45: Theming Support
The following content types need to support theming.

- main page branding
- tab view branding
- product page branding

### PR DESCRIPTION
**Pull request by arumsey**
## Issue

Fix #45

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
- After: https://theming--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
- After: https://theming--sgws-creative--hlxsites.hlx.page/bloominbrands/p/6

## 📝 Description:

- add theming to product templates
- CSS cleanup
- allow fragments to have multiple sections

### CODE REVIEW

File: styles/lazy-styles.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
   font-style: normal;
   font-weight: 400;
   font-display: swap;
+  unicode-range: U+0020-FB02;
```
Comment (tmathern) [inline styles/lazy-styles.css:8]: What is that? Why do we need it?

File: styles/styles.css [hunk 2]
```diff
@@ -25,28 +37,21 @@
...
-  --body-font-size-min: calc(var(--body-font-size) * 0.5);
-  --body-font-size-max: calc(var(--body-font-size));
+  --body-font-size-min: 0.75rem;
+  --body-font-size-max: var(--body-font-size);
 
   /* heading sizes */
-  --heading-font-size-xxl: calc(var(--heading-font-size) + 14px);
-  --heading-font-size-xl: calc(var(--heading-font-size) + 6px);
+  --heading-font-size-xxl: calc(var(--heading-font-size) + 0.875);
+  --heading-font-size-xl: calc(var(--heading-font-size) + 0.375);
```
Comment (tmathern) [inline styles/styles.css:50]: Is the unit missing?

File: styles/styles.css [hunk 3]
```diff
@@ -25,28 +37,21 @@
...
   /* body sizes */
-  --body-font-size-min: calc(var(--body-font-size) * 0.5);
-  --body-font-size-max: calc(var(--body-font-size));
+  --body-font-size-min: 0.75rem;
+  --body-font-size-max: var(--body-font-size);
 
   /* heading sizes */
-  --heading-font-size-xxl: calc(var(--heading-font-size) + 14px);
-  --heading-font-size-xl: calc(var(--heading-font-size) + 6px);
+  --heading-font-size-xxl: calc(var(--heading-font-size) + 0.875);
```
Comment (tmathern) [inline styles/styles.css:49]: Is the unit missing?

File: styles/styles.css [hunk 4]
```diff
@@ -328,7 +350,7 @@ svg#franklin-svg-sprite symbol {
   fill: #fff;
 }
 
-@media (min-width: 900px) {
+@media (min-width: 1200px) {
```
Comment (tmathern) [inline styles/styles.css:353]: I thought we want to get rid of all responsive breakpoints?

**Review by tmathern — APPROVED** [review]

File: styles/lazy-styles.css (hunk 1 above)
Comment (arumsey) [reply styles/lazy-styles.css]: I was attempting to improve font loading performance but it didn't help. Can be removed.

**Pull request merged**